include/librealsense2/dds/topics/device-info/device-info-msg.h

```diff
@@ -0,0 +1,38 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <string>
+#include "deviceInfoPubSubTypes.h"
+
+namespace librealsense {
+namespace dds {
+namespace topics {
+struct device_info
+{
+    std::string name;
+    std::string serial;
+    std::string product_line;
+    bool locked;
+
+    device_info()
```

Comment:
Why do we want an empty ctor?

Reply:
It's for convenience,
For instance, I need it in 2 different scopes.
first one set the values from the message,
second one use its values.
So I do..
```
device_info dev_info;
{
    dev_info = msg_raw;
}
...
{
  devices_map[...] = dev_info 
}
```

Reply:
Changed to `device_info() = default;`